lib/jsonDir.js

```diff
@@ -16,10 +17,30 @@ var collectJSONS = function (options) {
     if (files.length === 0) throw new Error('No JSON files found under \'options.jsonDir\': ' + options.jsonDir);
 
     function mergeJSONS(file) {
-        var cucumberJson = jsonFile.readFileSync(file);
+        let cucumberJson = jsonFile.readFileSync(file);
 
+        function trackScenarioRetries(scenarios) {
```

Comment:
It would be good to list this as a feature in the description of the PR, plus in the changelog.